src/Sulu/Component/Security/Authorization/AccessControl/AccessControlManager.php

```diff
@@ -87,7 +87,30 @@ public function getUserPermissions(SecurityCondition $securityCondition, UserInt
         $checkPermissionType = empty($objectPermissions);
 
         $securityContextPermissions = $this->getUserSecurityContextPermissions(
-            $securityCondition,
```

Comment:
Isn't it a possibility to create a `SecurityCondition` where ever you need this function call? Did this to avoid too many arguments...